src/libprojectM/Renderer/Renderer.cpp

```diff
@@ -76,10 +76,13 @@ Renderer::Renderer(int width, int height, int gx, int gy, BeatDetect* _beatDetec
 	title_fontURL(_titlefontURL), menu_fontURL(_menufontURL), presetURL(_presetURL)
 {
 	this->totalframes = 1;
-	this->lastTime = duration_cast<milliseconds>(system_clock::now().time_since_epoch());
-	this->currentTime = duration_cast<milliseconds>(system_clock::now().time_since_epoch());
+	this->lastTimeFPS = duration_cast<milliseconds>(system_clock::now().time_since_epoch());
+	this->currentTimeFPS = duration_cast<milliseconds>(system_clock::now().time_since_epoch());
+	this->lastTimeToast = duration_cast<milliseconds>(system_clock::now().time_since_epoch());
+	this->currentTimeToast = duration_cast<milliseconds>(system_clock::now().time_since_epoch());
```

Comment:
Maybe refactor `duration_cast<milliseconds>(system_clock::now().time_since_epoch())` to a function